### DISCUSSION TITLE
How to use Chakra UI  with remotion?

### DISCUSSION THREAD

**Discussion by winner106** [remotion-dev/remotion#1903] [Q&A] [answered]
Chakra UI: [https://chakra-ui.com/getting-started](https://chakra-ui.com/getting-started)

I have tried the following code , but it will mess up the preview page....

```jsx
export const Visx = () => {
	const {width, height} = useVideoConfig();
	return (
		<ChakraProvider theme={theme}>
			<ColumnChart width={width} height={height} />
		</ChakraProvider>
	);
};
```

```jsx
export const RemotionRoot: React.FC = () => {
	return (
		<>
			<Composition
				id="Visx"
				component={Visx}
				durationInFrames={150}
				fps={30}
				width={1920}
				height={1080}
			/>

		</>
	);
};
```

![image](https://user-images.githubusercontent.com/50519642/223764377-e5aa126c-8ea0-4c76-b041-f461cd075475.png)

**Comment by JonnyBurger** [comment, marked as answer]
Thanks for reporting this! I fix this in https://github.com/remotion-dev/remotion/pull/1908
